src/components/ZclAttributeManager.vue

```diff
@@ -39,7 +39,11 @@ limitations under the License.
         <q-tr
           :props="props"
           class="table_body"
-          v-if="!globalLists.includes(props.row.label)"
+          v-show="
+            (!globalLists.includes(props.row.label) &&
+              props.row.side === 'server') ||
```

Comment:
This should be matter only change. Other protocols such as Zigbee should not be affected by this. Is that the case here?

Reply:
it does do that `            (!globalLists.includes(props.row.label) &&
              props.row.side === 'server') ||
            !enableServerOnly`